docs/api/process.md

```diff
@@ -133,6 +134,11 @@ A `String` representing Electron's version string.
 A `Boolean`. If the app is running as a Windows Store app (appx), this property is `true`,
 for otherwise it is `undefined`.
 
+### `process.versions.contextId` _Readonly_
+
+A `String` (optional) representing unique ID of the current JavaScript context.
```

Comment:
it's only unique within the renderer i believe, we should note what scope it's unique in.

Reply:
isn't it globally unique?
https://github.com/electron/electron/blob/f114dcfd6e44a1b67cb83af40097fd49ea1f9239/shell/renderer/renderer_client_base.cc#L130-L138

Reply:
ah, you're right.